lib/integration/crud.ts

```diff
@@ -244,6 +246,28 @@ async function _getFormByStatus(
   return [];
 }
 
+const _getForms = async (templates: unknown[] = [], limit = 50, offset = 0): Promise<unknown[]> => {
```

Comment:
Although the intention of this function is not to stay long. A doc string would be much appreciated so we can remind ourselves as to the purpose of it

Reply:
I think the lack of docstrings in the rest of the file caused me to overlook this.  ;)